crates/bevy_app/src/app.rs

```diff
@@ -379,9 +379,10 @@ impl App {
         schedule: impl ScheduleLabel,
         systems: impl IntoSystemConfigs<M>,
     ) -> &mut Self {
+        let schedule: InternedScheduleLabel = (&schedule as &dyn ScheduleLabel).into();
```

Comment:
```suggestion
        let schedule = InternedScheduleLabel::from(&schedule as &dyn ScheduleLabel);
```

Not super important, but I think this syntax is preferable. Same for all of the other places this is used.

Reply:
I agree. Changed all places this was used.